interfaces/builtin/ppp.go

```diff
@@ -41,11 +41,6 @@ var pppConnectedPlugAppArmor = []byte(`
 @{PROC}/@{pid}/loginuid r,
 capability setgid,
 capability setuid,
-/sbin/resolvconf rix,
-/run/resolvconf** rw,
-/etc/resolvconf/** rw,
-/etc/resolvconf/update.d/* ix,
-/lib/resolvconf/* ix,
```

Comment:
It was actually a mistake to add those here. In the current setup it doesn't change much anyway as the only user of the ubuntu-core:ppp slot is the network-manager snap which then has resolvconf permissions already through the network-manager permanent slot.